homeassistant/components/huum/light.py

```diff
@@ -0,0 +1,61 @@
+"""Control for light."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from homeassistant.components.light import ColorMode, LightEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from .coordinator import HuumConfigEntry, HuumDataUpdateCoordinator
+from .entity import HuumBaseEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: HuumConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up light if applicable."""
+    coordinator = config_entry.runtime_data
+
+    # Light is configured for this sauna.
+    if coordinator.data.config >= 2:
+        async_add_entities([HuumLight(coordinator)], True)
+
+
+class HuumLight(HuumBaseEntity, LightEntity):
+    """Representation of a light."""
+
+    _attr_name = "Light"
+    _attr_supported_color_modes = set(ColorMode.ONOFF)
+    _attr_color_mode = ColorMode.ONOFF
+
+    def __init__(self, coordinator: HuumDataUpdateCoordinator) -> None:
+        """Initialize the light."""
+        super().__init__(coordinator)
+
+        self._attr_unique_id = f"{coordinator.config_entry.entry_id}_light"
```

Comment:
```suggestion
        self._attr_unique_id = f"{coordinator.config_entry.entry_id}"
```
those are unique per platform anyways, you could also just move the unique id from the climate to your base entity, so you don't need to set it here

Reply:
Question about this btw, isn't it good practice to compose a unique_id per entity anyways? Thats what I see being done in most components. Especially for sensors for example. Because what if I want to add another one in the future? At that moment I need to make them unique anyway, and if the first sensor does not have an addition and all following ones do, makes it a bit inconsistent?

Reply:
I'm not sure I understand your concern. If you're adding multiple sensors, you can just overwrite the unique id in the sensor class with `{entry_id}_{key}` making it consistent for all sensors.

Reply:
But when I do that, the sensor originally without the `{key}` will be recreated (and the old one unavailable) for everyone? Which is kinda annoying and easily avoided by putting the key there from the start.